crates/bevy_ecs/src/world/mod.rs

```diff
@@ -1409,6 +1561,9 @@ impl World {
         self.storages.tables.check_change_ticks(change_tick);
         self.storages.sparse_sets.check_change_ticks(change_tick);
         self.storages.resources.check_change_ticks(change_tick);
+        self.storages
```

Comment:
I wonder if we should be exhaustively matching on the `storages` field to ensure that when we add new ones we get a compile error bringing us here.